### ISSUE TITLE
Popover misplaced, twitches into right place on scroll due to larger image

### ISSUE THREAD

**Issue by tobsn** [twbs/bootstrap#19981]
**VISUAL:**
http://quick.as/14nyubljw

**DESCRIPTION:**
Popover is placed somewhere below the element (no other elements in that area, its dead space) - on the slightest scroll activity (popover most be visible) popover jumps into the right place.

**REPRODUCTION:**
https://jsfiddle.net/7jh1a4sv/1/

**Comment by twbs-lmvtfy** [comment]
Hi @tobsn!

You appear to have posted a live example (https://fiddle.jshell.net/7jh1a4sv/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 138, column 9 thru column 172: Bad value ```` cc-name ```` for attribute ```` autocomplete ```` on element ```` input ````.
* line 143, column 10 thru column 180: Bad value ```` cc-number ```` for attribute ```` autocomplete ```` on element ```` input ````.
* line 151, column 11 thru column 134: Bad value ```` cc-exp ```` for attribute ```` autocomplete ```` on element ```` input ````.
* line 160, column 11 thru column 151: Bad value ```` cc-csc ```` for attribute ```` autocomplete ```` on element ```` input ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by tobsn** [comment]
*facepalm

afaik those values are completely fine. either way they do not have anything to do with the issue.

**Comment by cvrebert** [comment]
Yeah, the HTML validator suffers from a bug: https://github.com/validator/validator/issues/268

**Comment by cvrebert** [comment]
With the irrelevant `autocomplete` attributes removed: https://jsfiddle.net/cvrebert/9r53hfnk/1/

**Comment by cvrebert** [comment]
Seems to be a bug in Tether. Seems as if it's not handling size changes caused by images loading in the tethered element. (My sympathy to the Tether team. The job they have is a difficult one.)

Example (which uses Tether & Drop and doesn't use Bootstrap's popover/tooltip code): https://jsfiddle.net/bw2v2q6s/8/

Steps to reproduce:
1. Open https://jsfiddle.net/bw2v2q6s/8/show/ in Chrome.
2. Open DevTools.
3. Go to the Network tab.
4. Check the "Disable cache" button.
5. Instead of "No throttling", select "GPRS" (or some other very slow option).
6. Click the "What is a CVC?" button.
7. Observe that, once the image finishes loading, the Drop is misplaced.
8. Observe that resizing the window triggers a position recalculation that correctly repositions the Drop.

**Comment by cvrebert** [comment]
Escalated to Tether: https://github.com/HubSpot/tether/issues/181

**Comment by cvrebert** [comment]
Reopening since Tether WontFix'd their issue.

**Comment by Avcajaraville** [comment]
Ok, so... what is the status on this issue then ?

**Comment by tobsn** [comment]
@Avcajaraville I assume someone will look at it when they have time. it's essentially a stupid problem where the image will misplace the popover once loaded... tether should be the party fixing that imho.

**Comment by Johann-S** [comment]
When this PR #22444 will be merged you will have access to `.popover('update')` method which will allow you to fix your issue @tobsn

**Comment by mdo** [comment]
Closing per above comment. Please open a new issue if the problem persists with Popper. <3
